joplin/base/factories/guide_page.py

```diff
@@ -0,0 +1,72 @@
+import factory
+import factory.fuzzy
+import wagtail_factories
+from django.utils.text import slugify
+from wagtail.core.models import Collection, Page
+from base.models import *
+from wagtail.core import blocks
+from .. import PageFactory, TextBlockFactory
+
+"""
+almost working
```

Comment:
this does work, but it atm relies on some existing objects in the database that I'd like to eventually also replace with factories

There is some goofiness due to M2M relationships with Clusterable Model, though